From 50cb4f595e388f1839a2d826b431793ae606f320 Mon Sep 17 00:00:00 2001
From: Anay Wadhera <anay1018@gmail.com>
Date: Mon, 24 Jan 2022 15:46:40 +0900
Subject: [PATCH 2/5] IconPackKaiAndroidOverlay: fix for carriers using 5 bars
 of signal

* this stuck out like an eyesore, fix it.

Change-Id: I97bbb0dc676967491ec0e26007d1059c91e22fc7
---
 .../drawable/ic_signal_cellular_0_5_bar.xml   | 22 +++++++++++++++++++
 .../drawable/ic_signal_cellular_1_5_bar.xml   | 22 +++++++++++++++++++
 .../drawable/ic_signal_cellular_2_5_bar.xml   | 22 +++++++++++++++++++
 .../drawable/ic_signal_cellular_3_5_bar.xml   | 22 +++++++++++++++++++
 .../drawable/ic_signal_cellular_4_5_bar.xml   | 22 +++++++++++++++++++
 .../drawable/ic_signal_cellular_5_5_bar.xml   | 22 +++++++++++++++++++
 6 files changed, 132 insertions(+)
 create mode 100644 packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_0_5_bar.xml
 create mode 100644 packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_1_5_bar.xml
 create mode 100644 packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_2_5_bar.xml
 create mode 100644 packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_3_5_bar.xml
 create mode 100644 packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_4_5_bar.xml
 create mode 100644 packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_5_5_bar.xml

diff --git a/packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_0_5_bar.xml b/packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_0_5_bar.xml
new file mode 100644
index 000000000000..214b9dd2b61e
--- /dev/null
+++ b/packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_0_5_bar.xml
@@ -0,0 +1,22 @@
+<!--
+   Copyright (C) 2020 The Android Open Source Project
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<vector android:height="24dp" android:viewportHeight="24" android:viewportWidth="24" android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+  <path android:fillAlpha="0.3" android:fillColor="@android:color/white" android:pathData="M 19.25 3 H 20.75 V 22 H 19.25 V 3 Z" android:strokeAlpha="0.3" android:strokeWidth="1"/>
+  <path android:fillAlpha="0.3" android:fillColor="@android:color/white" android:pathData="M 3.25 18 H 4.75 V 22 H 3.25 V 18 Z" android:strokeAlpha="0.3" android:strokeWidth="1"/>
+  <path android:fillAlpha="0.3" android:fillColor="@android:color/white" android:pathData="M 7.25 14 H 8.75 V 22 H 7.25 V 14 Z" android:strokeAlpha="0.3" android:strokeWidth="1"/>
+  <path android:fillAlpha="0.3" android:fillColor="@android:color/white" android:pathData="M 11.25 11 H 12.75 V 22 H 11.25 V 11 Z" android:strokeAlpha="0.3" android:strokeWidth="1"/>
+  <path android:fillAlpha="0.3" android:fillColor="@android:color/white" android:pathData="M 15.25 7 H 16.75 V 22 H 15.25 V 7 Z" android:strokeAlpha="0.3" android:strokeWidth="1"/>
+</vector>
diff --git a/packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_1_5_bar.xml b/packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_1_5_bar.xml
new file mode 100644
index 000000000000..c74c6ab9de4c
--- /dev/null
+++ b/packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_1_5_bar.xml
@@ -0,0 +1,22 @@
+<!--
+   Copyright (C) 2020 The Android Open Source Project
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<vector android:height="24dp" android:viewportHeight="24" android:viewportWidth="24" android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+  <path android:fillColor="@android:color/white" android:pathData="M 3.25 18 H 4.75 V 22 H 3.25 V 18 Z"/>
+  <path android:fillAlpha="0.3" android:fillColor="@android:color/white" android:pathData="M 7.25 14 H 8.75 V 22 H 7.25 V 14 Z" android:strokeAlpha="0.3" android:strokeWidth="1"/>
+  <path android:fillAlpha="0.3" android:fillColor="@android:color/white" android:pathData="M 19.25 3 H 20.75 V 22 H 19.25 V 3 Z" android:strokeAlpha="0.3" android:strokeWidth="1"/>
+  <path android:fillAlpha="0.3" android:fillColor="@android:color/white" android:pathData="M 11.25 11 H 12.75 V 22 H 11.25 V 11 Z" android:strokeAlpha="0.3" android:strokeWidth="1"/>
+  <path android:fillAlpha="0.3" android:fillColor="@android:color/white" android:pathData="M 15.25 7 H 16.75 V 22 H 15.25 V 7 Z" android:strokeAlpha="0.3" android:strokeWidth="1"/>
+</vector>
diff --git a/packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_2_5_bar.xml b/packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_2_5_bar.xml
new file mode 100644
index 000000000000..201792858551
--- /dev/null
+++ b/packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_2_5_bar.xml
@@ -0,0 +1,22 @@
+<!--
+   Copyright (C) 2020 The Android Open Source Project
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<vector android:height="24dp" android:viewportHeight="24" android:viewportWidth="24" android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+  <path android:fillColor="@android:color/white" android:pathData="M 3.25 18 H 4.75 V 22 H 3.25 V 18 Z"/>
+  <path android:fillColor="@android:color/white" android:pathData="M 7.25 14 H 8.75 V 22 H 7.25 V 14 Z"/>
+  <path android:fillAlpha="0.3" android:fillColor="@android:color/white" android:pathData="M 11.25 11 H 12.75 V 22 H 11.25 V 11 Z" android:strokeAlpha="0.3" android:strokeWidth="1"/>
+  <path android:fillAlpha="0.3" android:fillColor="@android:color/white" android:pathData="M 15.25 7 H 16.75 V 22 H 15.25 V 7 Z" android:strokeAlpha="0.3" android:strokeWidth="1"/>
+  <path android:fillAlpha="0.3" android:fillColor="@android:color/white" android:pathData="M 19.25 3 H 20.75 V 22 H 19.25 V 3 Z" android:strokeAlpha="0.3" android:strokeWidth="1"/>
+</vector>
diff --git a/packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_3_5_bar.xml b/packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_3_5_bar.xml
new file mode 100644
index 000000000000..40846745e3b0
--- /dev/null
+++ b/packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_3_5_bar.xml
@@ -0,0 +1,22 @@
+<!--
+   Copyright (C) 2020 The Android Open Source Project
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<vector android:height="24dp" android:viewportHeight="24" android:viewportWidth="24" android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+  <path android:fillColor="@android:color/white" android:pathData="M 3.25 18 H 4.75 V 22 H 3.25 V 18 Z"/>
+  <path android:fillColor="@android:color/white" android:pathData="M 7.25 14 H 8.75 V 22 H 7.25 V 14 Z"/>
+  <path android:fillColor="@android:color/white" android:pathData="M 11.25 11 H 12.75 V 22 H 11.25 V 11 Z"/>
+  <path android:fillAlpha="0.3" android:fillColor="@android:color/white" android:pathData="M 15.25 7 H 16.75 V 22 H 15.25 V 7 Z" android:strokeAlpha="0.3" android:strokeWidth="1"/>
+  <path android:fillAlpha="0.3" android:fillColor="@android:color/white" android:pathData="M 19.25 3 H 20.75 V 22 H 19.25 V 3 Z" android:strokeAlpha="0.3" android:strokeWidth="1"/>
+</vector>
diff --git a/packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_4_5_bar.xml b/packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_4_5_bar.xml
new file mode 100644
index 000000000000..1032619482d4
--- /dev/null
+++ b/packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_4_5_bar.xml
@@ -0,0 +1,22 @@
+<!--
+   Copyright (C) 2020 The Android Open Source Project
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<vector android:height="24dp" android:viewportHeight="24" android:viewportWidth="24" android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+  <path android:fillAlpha="0.3" android:fillColor="@android:color/white" android:pathData="M 19.25 3 H 20.75 V 22 H 19.25 V 3 Z" android:strokeAlpha="0.3" android:strokeWidth="1"/>
+  <path android:fillColor="@android:color/white" android:pathData="M 3.25 18 H 4.75 V 22 H 3.25 V 18 Z"/>
+  <path android:fillColor="@android:color/white" android:pathData="M 7.25 14 H 8.75 V 22 H 7.25 V 14 Z"/>
+  <path android:fillColor="@android:color/white" android:pathData="M 11.25 11 H 12.75 V 22 H 11.25 V 11 Z"/>
+  <path android:fillColor="@android:color/white" android:pathData="M 15.25 7 H 16.75 V 22 H 15.25 V 7 Z"/>
+</vector>
diff --git a/packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_5_5_bar.xml b/packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_5_5_bar.xml
new file mode 100644
index 000000000000..d022d9c2fd1b
--- /dev/null
+++ b/packages/overlays/IconPackKaiAndroidOverlay/res/drawable/ic_signal_cellular_5_5_bar.xml
@@ -0,0 +1,22 @@
+<!--
+   Copyright (C) 2020 The Android Open Source Project
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<vector android:height="24dp" android:viewportHeight="24" android:viewportWidth="24" android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+  <path android:fillColor="@android:color/white" android:pathData="M 19.25 3 H 20.75 V 22 H 19.25 V 3 Z"/>
+  <path android:fillColor="@android:color/white" android:pathData="M 3.25 18 H 4.75 V 22 H 3.25 V 18 Z"/>
+  <path android:fillColor="@android:color/white" android:pathData="M 7.25 14 H 8.75 V 22 H 7.25 V 14 Z"/>
+  <path android:fillColor="@android:color/white" android:pathData="M 11.25 11 H 12.75 V 22 H 11.25 V 11 Z"/>
+  <path android:fillColor="@android:color/white" android:pathData="M 15.25 7 H 16.75 V 22 H 15.25 V 7 Z"/>
+</vector>
\ No newline at end of file
-- 
2.25.1

